.github/workflows/tests_sources.yml

```diff
@@ -45,14 +45,41 @@ jobs:
         export OPENDR_HOME=$PWD
         pip install -r tests/requirements.txt
         python -m unittest discover -s tests
-    - name: Test API
-      if: ${{ matrix.os == 'ubuntu-20.04' && contains(github.event.pull_request.labels.*.name, 'test tools') }}
+  test-tools:
+    needs: cleanup-runs
+    if: ${{ contains(github.event.pull_request.labels.*.name, 'test tools') }}
+    strategy:
+      matrix:
+        package:
+          - engine
+          - utils
+          - perception/activity_recognition
+          - perception/compressive_learning
+          - perception/face_recognition
+          - perception/heart_anomaly_detection
+          - perception/multimodal_human_centric
+          - perception/object_tracking_2d
+          - perception/object_detection_3d
+          - perception/pose_estimation
+          - perception/speech_recognition
+          # - perception/object_tracking_3d
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v2
+      if: github.event.pull_request.draft == false
+    - name: Set up Python 3.8
+      if: github.event.pull_request.draft == false
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Test Tools
+      if: ${{ contains(github.event.pull_request.labels.*.name, 'test tools') }}
       run: |
-        ${{ matrix.DEPENDENCIES_INSTALLATION }}
+        sudo apt -y install clang-format-10 cppcheck
```

Comment:
Why do you need clang format and cppcheck for testing the functionality?
These tools should only be run in the `test-sources` job to check the code sources.

Reply:
You are right, it probably doesn't. I just replaced the `${{ matrix.DEPENDENCIES_INSTALLATION }}` step with the contents for ubuntu 20.04. This was done to keep the test steps the same as in the old version. 
The line was remove in commit dbd212689ece9ffd